phonenumbers.txt

```diff
@@ -1 +1,3 @@
-NTE5NDU1NTA1MA0KNDE2Nzc0MDEyMw0KNjQ3NTUyMDEyMw==
\ No newline at end of file
+5194555050
```

Comment:
Do not change my original txt file. It has to be able to parse multiple strings into an array.